packages/api/cms-api/src/builds/changes-checker.interceptor.ts

```diff
@@ -42,6 +49,8 @@ export class ChangesCheckerInterceptor implements NestInterceptor {
 
                             await this.buildsService.setChangesSinceLastBuild(scope);
                         }
+                    } else {
+                        await this.buildsService.setChangesSinceLastBuild("all");
```

Comment:
Should we add the warning in development here?

Reply:
I'm not sure if I fully understand your question, but yes, with the added warning, it's now the same behavior as it was before the user permissions related changes in contentScopeService. I believe it's a good idea to show this warning